.golangci.yaml

```diff
@@ -30,6 +30,7 @@ linters:
     - deadcode
     - varcheck
     - interfacebloat
+    - exhaustruct
```

Comment:
why disable this linter rule?

Reply:
it was causing too many linter issues as there are many paths that are not ignored. I will have a follow up to properly update the go version and linter